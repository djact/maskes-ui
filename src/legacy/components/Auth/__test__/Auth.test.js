import React from 'react'
import { shallow } from 'enzyme'
import { Auth } from '..'

describe('Test Auth Component', () => {
    it('should shallow render correctly Auth component', () => {
        const AuthComponent = shallow(<Auth />)
        expect(AuthComponent).toMatchSnapshot()
    })
})
