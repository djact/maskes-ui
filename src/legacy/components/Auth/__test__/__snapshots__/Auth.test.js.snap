// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Auth Component should shallow render correctly Auth component 1`] = `
<AuthModal
  isLogin={true}
  onChange={[Function]}
  onSubmit={[Function]}
  switchMode={[Function]}
/>
`;
